core/settings.py

```diff
@@ -139,6 +160,8 @@
 DATA_LOCATION = CURRENT_PATH + '/media/data'
 DATA_URL = '/agentexoplanet/media/data'
 
+ALLOWED_HOSTS = ['*']
+
```

Comment:
The `Templates` have changed in Django 1.9 so it would be good to switch over to the [new way of doing this](https://docs.djangoproject.com/en/1.8/ref/templates/upgrading/#the-templates-settings).